api/environments/identities/views.py

```diff
@@ -55,11 +142,23 @@ def get_environment_from_request(self):
 
     def perform_create(self, serializer):
         environment = self.get_environment_from_request()
+        if environment.project.enable_dynamo_db:
+            return Identity.put_item_dynamodb({**serializer.data, **self.kwargs})
         serializer.save(environment=environment)
 
+    def perform_destroy(self, instance):
+        environment = self.get_environment_from_request()
+        if environment.project.enable_dynamo_db:
+            Identity.delete_in_dynamodb(instance.composite_key)
+        else:
```

Comment:
Again, this `else` is unnecessary. You should almost be able to write all your code in python without writing a single `else` (being hyper optimistic there but it's _theoretically_ possible).